share/completions/nvim.fish

```diff
@@ -1 +1,59 @@
-complete -c nvim -w vim
+# Options copied from vim.fish
+complete -c nvim -s c -r -d 'Execute Ex command after the first file has been read'
+complete -c nvim -s S -r -d 'Source file after the first file has been read'
+complete -c nvim -l cmd -r -d 'Execute Ex command before loading any vimrc'
+complete -c nvim -s i -r -d 'Set the shada file location'
+complete -c nvim -s o -r -d 'Open horizontally split windows for each file'
+complete -c nvim -s O -r -d 'Open vertically split windows for each file'
+complete -c nvim -s p -r -d 'Open tab pages for each file'
+complete -c nvim -s q -r -d 'Start in quickFix mode'
+complete -c nvim -s r -r -d 'Use swap files for recovery'
+complete -c nvim -s t -xa '(__fish_vim_tags)' -d 'Set the cursor to tag'
```

Comment:
You can't use `__fish_vim_tags` here because there is no guarantee the function is loaded. A function is automatically loaded from a file if the file name matches the function name (and the file containing the function is in the function search path). But `__fish_vim_tags` is written in `vim.fish`.

You would need to either move `__fish_vim_tags` to another file, copy it (with a new name) to this file, or (best option, imho) add the following at the top of `nvim.fish`:

```fish
type -q __fish_vim_tags || source (status dirname)/vim.fish
```

to load the functions defined in `vim.fish` before running the completion script (if the vim completions script hasn't yet been loaded because the user tried to complete `vim`).